### PR TITLE
Feat: Add colour and bg_colour functions

### PR DESCRIPTION
**Pull request by brettkolodny**

### CODE REVIEW

File: src/gleam_community/ansi.gleam [hunk 1]
```diff
@@ -93,6 +97,7 @@ import gleam/bitwise
 import gleam/int
 import gleam/list
 import gleam/string
+import gleam_community/colour.{Colour} as gc_colour
```
Comment (hayleigh-dot-dev) [inline src/gleam_community/ansi.gleam:100]: I don't know how much you want my personal opinions to factor into review, but personally I don't alias imports _ever_ unless there's a clash (eg the `gleam_community/maths` stuff clashing with stdlib).

Comment (hayleigh-dot-dev) [inline src/gleam_community/ansi.gleam:100]: I don't know how much you want my personal opinions to factor into review, but personally I don't alias imports _ever_ unless there's a clash (eg the `gleam_community/maths` stuff clashing with stdlib).

**Review by hayleigh-dot-dev — APPROVED** [review]
I'm not gonna block merging based on an import alias, up to you whether you want to do anything with that or not. Good to release :)

File: src/gleam_community/ansi.gleam (hunk 1 above)
Comment (brettkolodny) [reply src/gleam_community/ansi.gleam:100]: I generally agree, for this the alias comes from the `colour` function also having a parameter named `colour` which then causes the clash. I figured better to alias the import than come up with a different parameter that may be more confusing.

**Pull request merged**